tests/test_asyncio/test_cluster.py

```diff
@@ -1443,7 +1443,7 @@ async def test_client_trackinginfo(self, r: RedisCluster) -> None:
         node = r.get_primaries()[0]
         res = await r.client_trackinginfo(target_nodes=node)
         assert len(res) > 2
-        assert "prefixes" in res
+        assert "prefixes" in res or b"prefixes" in res
```

Comment:
`assert res in ["prefixes", b"prefixes"]`
etc?

Reply:
Why it is better?

Reply:
IMO `"prefixes" in res or b"prefixes" in res` is more readable and understandable, and since it's only 2 options I don't see why using a map and a lambda is better